app/_data/tproxy.yaml

```diff
@@ -0,0 +1,109 @@
+config:
+  redirect:
+    inbound:
+      excludePorts: &excludePortsInbound
+        path: runtime.kubernetes.injector.redirect.inbound.excludePorts
+        flag: --exclude-inbound-ports
+        env: KUMA_TRANSPARENT_PROXY_REDIRECT_INBOUND_EXCLUDE_PORTS
+        runtime:
+          path: sidecarTraffic.excludeInboundPorts
+          env: KUMA_RUNTIME_KUBERNETES_SIDECAR_TRAFFIC_EXCLUDE_INBOUND_PORTS
+      excludePortsForIPs: &excludePortsForIPsInbound
+        path: runtime.kubernetes.injector.redirect.inbound.excludePortsForIPs
+        flag: --exclude-inbound-ips
```

Comment:
Is this correct? The path mentions `ports` and the flag does not 🤔.

Reply:
it's correct, internally it's called excludePortsForIPs but this flag is configuring this field